about.html

```diff
@@ -1,3 +1,5 @@
+<h2>Code of Conduct</h2>
```

Comment:
I think "General Behaviour" would be a better title here.

Reply:
Also, this heading should be placed below the first paragraphs.

Reply:
I don't think that the **Publication rules** have anything to do with "being excellent to each other", especially not the part about third-party-content.

Reply:
Changed the title from "Code of Conduct" to "General behaviour" as suggested.